src/common/socket.cpp

```diff
@@ -531,17 +531,26 @@ wxSocketImpl *wxSocketImpl::Accept(wxSocketBase& wxsocket)
     ReenableEvents(wxSOCKET_INPUT_FLAG);
 
     if ( fd == INVALID_SOCKET )
+    {
+	GetLastError();
```

Comment:
Minor, but wrong indentation is used here and elsewhere below, please check the diff.